src/cobalt.rs

```diff
@@ -237,8 +238,13 @@ pub fn build(config: &Config) -> Result<()> {
     }
 
     // copy all remaining files in the source to the destination
+    // compile SASS along the way
     {
         info!("Copying remaining assets");
+        let mut sass_opts = SassOptions::default();
+        //  FIXME: make this a config option
+        sass_opts.include_paths =
+            vec![source.join("_sass").into_os_string().into_string().unwrap()];
```

Comment:
Adding an option for sass_dir would imply to me that that is where all the sass files should live but the implementation allows them to live anywhere.  Thoughts on which direction this should go?

Reply:
In that case, maybe we should call the config option

```
sass:
    import_dir: _sass
    style: compressed
```

It'll make the semantics more clear I feel.